sympy/utilities/codegen.py

```diff
@@ -666,6 +707,8 @@ def routine(self, name, expr, argument_sequence, global_vars):
 
         return Routine(name, arg_list, return_val, local_vars, global_vars)
 
+
+
```

Comment:
The convention is one blank line between methods (and two blank lines between e.g. classes)